doc/COMPILE.md

```diff
@@ -512,7 +512,7 @@ If both options are provided the value passed via CMake takes precedence.
 
 [google test]: https://github.com/google/googletest
 
-It is recommended that you browse through all of the options using `ccmake`.
```

Comment:
This seems to me to be the correct form already and might not need to be changed.

Reply:
Thank you for the feedback!
The article https://jakubmarian.com/is-it-all-the-or-all-of-the-in-english contains the following statement about this topic: _Having said that, the variant without “of” is significantly more common than the variant with “of”, to such a degree that the variant with “of” might be considered unnatural (or colloquial) by some native speakers in certain contexts. My advice is: If in doubt, do not use “of” between “all” and another determiner._

In the posting  # 18 in https://forum.wordreference.com/threads/all-versus-all-of.605984/post-15271893, "Garner's Modern American usage (3rd edition)" is cited. That citation contains the part: _The more formal construction is to omit of_.

Having stated that, it's definitely not an important or even necessary change and I think that both variants are totally ok.